_plugins/version.rb

```diff
@@ -0,0 +1,52 @@
+# Aggregate the En lesson versions together
+Jekyll::Hooks.register :site, :post_read do |site|
```

Comment:
@doomspork Sounds good. Then what should we do next? Are we going to merge the versioning updates in the current state (this PR and #962)?